src/Chapter09.Tests/Listing09.14.RecordDeconstructors.Tests.cs

```diff
@@ -0,0 +1,21 @@
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_14.Tests;
+
+using AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_14;
+
+
+[TestClass]
+public class DeconstructorTest
+{
+    [TestMethod]
+    public void DeconstructorValues_MatchAngleValues()
+    {
+        // The constructor is generated using positional parameters
+        Angle angle = new(90, 10, 10);
+
+        angle.Deconstruct(out int degrees, out int minutes, out int seconds);
+
+        Assert.AreEqual(degrees, degrees);
+        Assert.AreEqual(minutes, minutes);
+        Assert.AreEqual(seconds, seconds);
```

Comment:
Something is still off here. You are comparing values against themselves. Typically you will want to have the "expectec" (first parameter) be the same value from the "arrange" portion of the test

Reply:
This one still doesn't look to be resolved. To be clear, the issue is `Assert.AreEqual(degree, degrees);` is just comparing `degrees` with itself (where it will always be true). We should be comparting it against the values that were passed above. Something more like: `Assert.AreEqual(90, degrees)`